static/index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang='en'>
+<head>
+  <meta charset='UTF-8'>
+  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
+  <title>NEAR Data Server by FASTNEAR</title>
+  <style>
+    html {
+      max-width: 70ch;
+      padding: 3em 1em;
+      margin: auto;
+      line-height: 1.75;
+      font-size: 1.25em;
+    }
+
+    #chart {
+      width: 800px;
+      height: 400px;
+      margin: 40px auto;
+    }
+  </style>
+</head>
+<body>
+<h1>NEAR Data Server by FASTNEAR</h1>
+<p>For more information, visit <a href='https://github.com/fastnear/neardata-server/'>GitHub</a></p>
+
+<h2>LIVE LATENCY TEST</h2>
+<div id="latency">
+
+  <script src="https://cdn.jsdelivr.net/npm/chart.js@3.7.1/dist/chart.min.js"></script>
```

Comment:
Haven't seen this in a while, and forgot how simple and clean it is. If it's able to pull in any ol' static HTML file, I'm already thinking of how these might be generated/updated on the server. /hack-it-friday-thought

Reply:
This basic chart example was actually generated by demo.fastnear.ai 
Then just modified it to feed real data